orchagent/bulker.h

```diff
@@ -104,6 +188,54 @@ namespace std
             return seed;
         }
     };
+
+    template <>
+    struct hash<sai_inbound_routing_entry_t>
+    {
+        size_t operator()(const sai_inbound_routing_entry_t& a) const noexcept
+        {
+            size_t seed = 0;
+            boost::hash_combine(seed, a.switch_id);
```

Comment:
Do we need to include the switch ID here? @prsunny I was under the impression that the switch ID is set at init time and never changes

Reply:
Switch ID does not change, but I kept the existing convention used for sai_route_entry, sai_fdb_entry etc for consistency.